faq/migration-tidb-faq.md

```diff
@@ -124,5 +124,5 @@ You can configure concurrent GC to increase the speed of reclaiming storage spac
 {{< copyable "sql" >}}
 
 ```sql
-update mysql.tidb set VARIABLE_VALUE="3" where VARIABLE_NAME="tikv_gc_concurrency";
-```
+SET GLOBAL tidb_gc_concurrency = 3;
```

Comment:
Looks like this faq is not valid since 3.0.... Can we just remove this...

Reply:
I will delete it from here, but rewrite it in sql-faq.md. It still comes up as a question.